test/verification_script.js

```diff
@@ -163,18 +163,18 @@ contract("Verification checks", function (accounts) {
       })
     })
   })
-  // describe("Brackets' deployed bytecode coincides with that of a Gnosis Safe proxy", async () => {
-  //   it("throws if bytecode differs", async () => {
-  //     const masterSafe = await GnosisSafe.at(await deploySafe(gnosisSafeMasterCopy, proxyFactory, [safeOwner], 1))
-  //     const evilProxy = await EvilGnosisSafeProxy.new(GnosisSafe.address)
-  //     const evilSafe = await GnosisSafe.at(evilProxy.address)
-  //     await evilSafe.setup([masterSafe.address], "1", ZERO_ADDRESS, "0x", ZERO_ADDRESS, ZERO_ADDRESS, "0", ZERO_ADDRESS)
-  //     await assert.rejects(verifyCorrectSetup([evilProxy.address], masterSafe.address), {
-  //       message: `Bytecode at bracket ${evilProxy.address} does not agree with that GnosisSafeProxy v1.1.1`
-  //     })
-  //     // TODO - this test only checks against bad proxy, but not agains bad safe.
-  //   })
-  // })
+  describe("Brackets' deployed bytecode coincides with that of a Gnosis Safe proxy", async () => {
+    it("throws if bytecode differs", async () => {
+      const masterSafe = await GnosisSafe.at(await deploySafe(gnosisSafeMasterCopy, proxyFactory, [safeOwner], 1))
+      const evilProxy = await EvilGnosisSafeProxy.new(GnosisSafe.address)
+      const evilSafe = await GnosisSafe.at(evilProxy.address)
+      await evilSafe.setup([masterSafe.address], "1", ZERO_ADDRESS, "0x", ZERO_ADDRESS, ZERO_ADDRESS, "0", ZERO_ADDRESS)
+      await assert.rejects(verifyCorrectSetup([evilProxy.address], masterSafe.address), {
+        message: `Bytecode at bracket ${evilProxy.address} does not agree with that of a Gnosis Safe Proxy v1.1.1`,
+      })
+      // TODO - this test only checks against bad proxy, but not against a bad safe.
```

Comment:
This comment was already there. I'd say we don't need to do further checks, since there is no fundamental reason why this check should not pass with a bad proxy but pass with a "bad Safe". Is it fine if I delete this TODO?